src/components/NewMessageModal.vue

```diff
@@ -181,6 +185,19 @@ export default {
 				showError(t('mail', 'Could not discard message'))
 			}
 		},
+		convertEditorBody(composerData) {
+			if (composerData.editorBody !== null) {
+				return composerData.editorBody
+			}
+			if (composerData.body === null) {
+				return ''
+			}
+			if (composerData.body.format !== 'html') {
+				return toHtml(composerData.body).value
```

Comment:
https://github.com/nextcloud/mail/blob/213c25884ed2162e8d25e8ca3b6a6c6255170ea1/src/util/text.js#L199-L201 returns the identical object.

Reply:
ah, thank you

Reply:
But, i still cannot remove it. This is still needed, if backend give a plain text, this should be explicitly converted to html. Otherwise we would run into problem described here https://github.com/nextcloud/mail/pull/6372#issuecomment-1144910893

Reply:
I think my intent was not clear. I did not suggest dropping the conversion, only your if-else condition.

```js
			if (composerData.body.format !== 'html') {
				return toHtml(composerData.body).value
			} else {
				 return composerData.body.value
			}
```

should be

```js
			return toHtml(composerData.body).value
```

If this doesn't make sense then leave the code as is.